packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -985,16 +984,11 @@ export default class Meeting extends StatelessWebexPlugin {
         correlationId: this.correlationId,
         userId: this.userId,
         deviceId: this.deviceUrl,
-        orgId: this.orgId
+        orgId: this.orgId,
+        locusUrl: this.webex.internal.services.get('locus')
       };
 
-      // If locus is passed take the value from there else use meeting object
-      if (locus) {
```

Comment:
this shouldn't exist anymore, `locus` should be replaced with `locusUrl` and `locusInfo`